testsuite/gnat2goto/tests/package_use_type_N_Use_Type_Clause/Use_Type_Clause.adb

```diff
@@ -1,8 +1,21 @@
-with External_Types; use type External_Types.New_Integer;
+--  'use type' clause makes visible only the operators associated with a type
+--  (NB but not the type itself, this must still be fully qualified
+with Count_Types; use type Count_Types.Count;
+
+--  'use all type' clause makes visible both the operators and basic operations
```

Comment:
:pick: By "basic" operations do you mean [primitive operations](https://en.wikibooks.org/wiki/Ada_Programming/Object_Orientation#Primitive_operations)? That's what they're usually called in Ada related discussions so if that's the case we should use that lingo here as well.

Reply:
Ah yes. Hmm - I think 'basic' would refer to those inherited according to Ada lingo? 
Mixup. Fixed.